enterprise/backend/src/metabase_enterprise/action_v2/actions.clj

```diff
@@ -46,6 +50,12 @@
   (-> (assoc input :database (:id (actions/cached-database-via-table-id (:table-id input))))
       (update :row #(update-keys % u/qualified-name))))
 
+(defmethod actions/action-arg-map-schema :data-editing/undo [_action] :map)
+(defmethod actions/action-arg-map-schema :data-editing/redo [_action] :map)
+
+(defmethod actions/normalize-action-arg-map :data-editing/undo [_action _input] {})
+(defmethod actions/normalize-action-arg-map :data-editing/redo [_action _input] {})
```

Comment:
Idea: I wonder whether we should use `default-mapping` to capture the scope as a first class `input` and then use `normalize-action-arg-map` to normalize the scope.

This allows the action itself to be a little simpler, and makes things a bit more flexible (e.g. would allow us to create a custom "undo button" inside a data-app.

Reply:
We could do this as a follow up PR, just floating the idea